planet/src/main/webapp/index.html

```diff
@@ -41,10 +50,45 @@ <h1>Get ready to
             </div>
         </div>
     </div>
-    <div class="section white middle" id="next">
-        <div class="row container">
-            <p>Insert text here. To be determined.</p>
+    <div id="next">              
+    <a id="up" href=# class="waves-effect waves-light btn-floating btn red accent-2"><i class="material-icons">expand_less</i></a>        
+        <div class="row card-container center">
+            <div class="column center">
+                <div class="card">
+                    <h3>Step 1:</h3>
+                    <h3>Create a profile</h3>
+                    <hr/>
+                    <p>Sign up or log in to get started. Create a profile and connect with other travellers. View the travellers visiting the same places as you.</p>
+                    <a class="go waves-effect waves-light btn red accent-2">GO</a>
+                </div>                
+            </div>
+            <div class="column center">
+                <div class="card">
+                    <h3>Step 2:</h3>
+                    <h3>Search Destinations</h3>
+                    <hr/>
+                    <p>Navigate to the Planet home page and start searching and saving destinations. View your searched locations on our map and browse through a list
+                        with details about each place.</p>
+                    <a href="places.html" class="go waves-effect waves-light btn red accent-2">GO</a>
+                </div>            
+            </div>
+            <div class="column center">
+                <div class="card">
+                    <h3>Step 3:</h3>
+                    <h3>Generate an Itinerary</h3>
+                    <hr/>
+                    <p>On the Itinerary page, pick and choose from your saved places to generate an itinerary. Enter in an estimated visit time and create custom 
+                        events to add. Choose between optimized and ordered itineraries to generate.</p>
+                    <a href="itinerary.html" class="go waves-effect waves-light btn red accent-2">GO</a>
```

Comment:
What happens if the user does not have any places saved and they click through to here? Will they be ushered back to the places page?

Reply:
The users will still be able to see the itinerary page even when not signed it! I think I will work with Alice to display a message that prompts the user to sign in to actually be able to access saved places and create an itinerary